django_auth_ldap/backend.py

```diff
@@ -913,7 +913,16 @@ def is_member_of(self, group_dn):
         # If we have self._group_dns, we'll use it. Otherwise, we'll try to
         # avoid the cost of loading it.
         if self._group_dns is None:
-            is_member = self._group_type.is_member(self._ldap_user, group_dn)
+            try:
+                is_member = self._group_type.is_member(self._ldap_user, group_dn)
+            except ldap.NO_SUCH_OBJECT as e:
```

Comment:
Why not add the `NO_SUCH_OBJECT` exception handling as part of https://github.com/django-auth-ldap/django-auth-ldap/blob/3e23f946590232eafabfbf96bf32fc6977f50386/django_auth_ldap/config.py#L443 and https://github.com/django-auth-ldap/django-auth-ldap/blob/3e23f946590232eafabfbf96bf32fc6977f50386/django_auth_ldap/config.py#L452?

Reply:
We can do that but, then either we have to log a warning for all three or add another except separate from `UNDEFINED_TYEP `and `NO_SUCH_ATTRIBUTE`

Reply:
I was thinking to add an except separate from `UNDEFINED_TYPE` and `NO_SUCH_ATTRIBUTE`.
```python
except (ldap.UNDEFINED_TYPE, ldap.NO_SUCH_ATTRIBUTE, ldap.NO_SUCH_OBJECT)
```

The query could not complete, user is not a member of the group (either because the object is not of a known type, the user does not have a membership attribute or the group in question does not exist).
WDYT?

Reply:
That sounds good I will commit the changes tomorrow.

Reply:
Sorry for my delay I commited the changes.

Reply:
No worries, thanks for pushing the changes!